### PR TITLE
Clarify MFA instructions

### PR DESCRIPTION
**Pull request by snowskeleton**
a user in Discord recently had trouble comprehending the MFA instructions

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -55,9 +55,9 @@ For help, or just to chat with fellow mintapi users, please [join us on Discord]
 
 ## Complete Setup
 ### MFA Authentication Methods
-You can handle MFA in one of two ways: email or TOTP
+You can handle MFA in one of four ways: text, voice, email, or TOTP
 (**T**ime-based **O**ne-**T**ime **P**assword). TOTP is strongly recommended.
-While Mint supports authentication via Voice, `mintapi` currently does not.
+`mintapi` does not support automated MFA through voice or text.
```
Comment (burkematthew) [inline README.md:60]: We do support text-based MFA.

File: README.md [hunk 2]
```diff
@@ -55,9 +55,9 @@ For help, or just to chat with fellow mintapi users, please [join us on Discord]
 
 ## Complete Setup
 ### MFA Authentication Methods
-You can handle MFA in one of two ways: email or TOTP
+You can handle MFA in one of four ways: text, voice, email, or TOTP
```
Comment (burkematthew) [inline README.md:58]: I think this statement should indicate that MINT provides these as options for MFA.  It's a little confusing to say "you can handle MFA in on of four ways", but then say, you can't do this or that one, though, here.

File: README.md [hunk 3]
```diff
@@ -73,14 +73,17 @@ Prior to v2.0, `mfa_method` is always required.
...
-**copy the manual setup code** that appears next to it. 
-**BE CAREFUL WHERE YOU STORE THIS**, as anyone with it will be able to take over your Mint account.
+Token MFA is only allowed as a secondary option,
+so first enable one of the other options (text, email, voice).
+Select the *Authenticator app* option.
+When you get to the part where you see the QR code,
+copy the **manual setup code** that appears next to it.
 This is the token you pass to `mfa_token` in either the python api or from the command line.
+Complete the setup process by providing a generated code from your authenticator app when requested.
+**BE CAREFUL WHERE YOU STORE THIS**, as anyone with it will be able to take over your Mint account.
```
Comment (burkematthew) [inline README.md:86]: While here, can you convert this markdown to a list?

**Review by burkematthew — CHANGES_REQUESTED** [review]

File: README.md (hunk 3 above)
Comment (snowskeleton) [reply README.md]: Converted to list.

File: README.md (hunk 1 above)
Comment (snowskeleton) [reply README.md]: Updated

File: README.md (hunk 2 above)
Comment (snowskeleton) [reply README.md]: Changed the wording to draw a finer distinction between what is offered by Mint and what mintapi accepts.

**Comment by snowskeleton** [comment]
It occurs to me that, if mintapi supports text MFA, there should be a section describing the process. I'm not familiar with that workflow, so someone else should be the one to write it. I imagine it could be added as an Option 3 in this same section.

**Review by burkematthew — APPROVED** [review]
Thanks for the doc cleanup!  Always appreciated!

**Pull request merged**